src/main/java/org/radarbase/appserver/service/scheduler/quartz/MessageJob.java

```diff
@@ -61,13 +60,13 @@ public class MessageJob implements Job {
   private final transient UserService userService;
```

Comment:
i presume this is not needed in this class anymore

Reply:
Indeed, I removed this reference.